mne/gui/_coreg.py

```diff
@@ -56,6 +56,15 @@ class CoregistrationUI(HasTraits):
         If True, display the EEG channels. Defaults to True.
     orient_glyphs : bool
         If True, orient the sensors towards the head surface. Default to False.
+    scale_by_distance : bool
+        If True, scale the sensors based on their distance to the head surface.
+        Defaults to True.
+    project_eeg : bool
+        If True, project the EEG channels onto the head surface. Defaults to
+        True.
```

Comment:
It should probably default to False. They are actually useful for coreg. Projection to surface is mostly for understanding how they will appear during forward modeling, so it's more like extra information if people are curious rather than being super useful during the coreg process itself.